### PR TITLE
Resume building dolfin page on CI

### LINKED ISSUE #30: restore dolfinx tutorial notebook builds
#27 stopped building the fenics notebook because it was crashing. We should re-enable it in a PR so we can more easily debug what's going wrong before merging it back into main by removing the `exclude_patterns` config from `_config.yml`:

```diff
--- a/_config.yml
+++ b/_config.yml
@@ -10,9 +10,6 @@ logo: logo.png
 execute:
   execute_notebooks: force
   timeout: 1800
-  exclude_patterns:
-    # temporarily skip dolfinx rebuild so we can figure it out
-    - notebooks/dolfinx_MPI_tutorial*
```

### PR DESCRIPTION
**Pull request by hherlyng**
Trying to run with dolfinx notebook to see if it builds properly

closes #30

### CODE REVIEW

**Comment by hherlyng** [comment]
also added petsc4py as dependency prior to this, but notebook still does not build

File: _config.yml [hunk 1]
```diff
@@ -10,9 +10,9 @@ logo: logo.png
 execute:
   execute_notebooks: cache
   timeout: 1800
-  exclude_patterns:
+  #exclude_patterns:
     # temporarily skip dolfinx rebuild so we can figure it out
-    - notebooks/dolfinx_MPI_tutorial*
+   # - notebooks/dolfinx_MPI_tutorial*
```
Comment (minrk) [inline _config.yml]: We can just remove these lines, rather than commenting out

**Review by minrk — COMMENTED** [review]
The engines are still [crashing](https://github.com/scientificcomputing/mpi-tutorial/pull/35/checks#step:7:73), it's unclear why.

Could it be out of memory?

File: _config.yml (hunk 1 above)
Comment (hherlyng) [reply _config.yml]: All right. Yeah maybe there is some memory error related to the pyvista plotting. But I just tried running it now with the latest docker image and it fails to build, so I think there is still some dependency missing

Comment (hherlyng) [reply _config.yml]: (because it works when I run it with my personal conda environment)

Comment (jorgensd) [reply _config.yml]: I guess you might be missing the panel backend (if we add the PYVISTA_JUPYTER_BACKEND=panel to the CI). We should Also set PYVISTA_OFF_SCREEN=False

Comment (hherlyng) [reply _config.yml]: Good catch! Added panel to the dependencies in environment.yml.

How can I set those environment variables for the docker image? I've tried several things including changing _config.yml and environment.yml, but I can't seem to get it working right.

Comment (jorgensd) [reply _config.yml]: See L23 and L24 in https://github.com/jorgensd/dolfinx-tutorial/blob/main/.github/workflows/book_stable.yml#L23

**Pull request merged**